src/summary/metadata/TreatmentOptionsSection.jsx

```diff
@@ -21,17 +21,71 @@ export default class TreatmentOptionsSection extends MetadataSection {
             data: [
                 {
                     name: "",
+                    // TO DO: need to change filters to support a function. getTreatmentCriteria below that gets patient and condition as arguments
+                    // eventually, the service API and implementation will need to support this call to figure out the supported criteria based on the condition
+                    // filterFunction: this.getTreatmentCriteria
+                    filters: [
+                        { id:"ageAtDiagnosis", name: "Age at diagnosis", servicePropertyName: "ageAtDiagnosis", category: "Demographics", value: true, 
+                                propertyValueFunction: (patient, condition) => { return patient.getAgeAsOf(new Date(condition.getDiagnosisDate())) } },
+                        { id:"gender",name: "Gender", servicePropertyName: "gender", category: "Demographics", value: true, 
+                                propertyValueFunction: (patient, condition) => { return patient.getGender() } },
+                        { id:"race",name: "Race", servicePropertyName: "race", category: "Demographics", value: true,
+                                propertyValueFunction: (patient, condition) => { return this.toFirstLetterCapital(patient.getPatient().race) } },
+                        { id:"kit",name: "KIT", servicePropertyName: "kit", category: "Genetics", value: true,
+                                propertyValueFunction: (patient, condition) => { return  condition.getGeneticMutationValue('KIT', patient) } },
+                        { id:"pdgfra", name: "PDGFRA", servicePropertyName: "pdgfra", category: "Genetics", value: true,
+                                propertyValueFunction: (patient, condition) => { return  condition.getGeneticMutationValue('PDGFRA', patient) } },
+                        { id:"grade",name: "Grade", servicePropertyName: "dxGrade", category: "Pathology", value: true,
+                                propertyValueFunction: (patient, condition) => { return condition.getMostRecentHistologicalGrade().getGradeAsSimpleNumber() } },
+                        { id:"stage",name: "Stage", servicePropertyName: "stage", category: "Pathology", value: true,
+                                propertyValueFunction: (patient, condition) => { return condition.getMostRecentStaging().stage } },
+                        { id:"surgery",name: "Surgery", servicePropertyName: "surgery", category: "Past Treatment", value: true,
+                                propertyValueFunction: (patient, condition) => { return condition.hasPastTreatment('C0851238', patient) } }
+                    ],
                     itemsFunction: this.getTreatmentData
                 }
             ]
         };
     }
 
-    getTreatmentData = (patient, condition, subsection) => {
+    // getTreatmentCriteria = (patient, condition) => {
```

Comment:
A lot of commented out code in this file(lines 51-62 and lines 69 - 88.  Do we want to keep these comments in here for later use?

Other than that, the criteria options look great!

Reply:
I was thinking to keep that code as we will need to ask the service for criteria that make sense given a condition.

Reply:
ok cool I will approve and merge then!